content/src/content/jcr_root/apps/core/wcm/components/embed/v1/embed/README.md

```diff
@@ -0,0 +1,104 @@
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+Embed (v1)
+====
+Embed component written in HTL that allows third-party widgets (e.g. chatbots, lead generation forms, social media posts, social pixels, videos etc.) to be added to a page.
+
+## Features
+* The following input types are supported:
+    * **URL** - an author is able to paste a URL of a widget to embed. URLs are checked against registered processors for a match. The following URL processors are provided:
+        * **oEmbed** - generic oEmbed URL processor with out-of-the-box configurations for Facebook Post, Instagram, SoundCloud, Twitter and YouTube.
+        * **Pinterest** - processes Pinterest URLs. 
+    * **Embeddable** - an author is able to select from pre-configured trusted embeddables. Embeddables can be parameterized and may include unsafe tags.
+    * **HTML** - an author is able to enter free-form HTML. HTML is restricted to safe tags only.
+* Each input type can be disabled by a template author.
+* For the embeddable type, the embeddables that are allowed to be selected in the edit dialog can be configured by a template author.
+
+### Component Policy Configuration Properties
+The following configuration properties are used:
+
+1. `./urlDisabled` - defines whether or not URL input is disabled in the edit dialog.
+2. `./embeddablesDisabled` - defines whether or not embeddables are disabled in the edit dialog.
+1. `./htmlDisabled` - defines whether or not free-form HTML input is disabled in the edit dialog.
+3. `./allowedEmbeddables` - defines the embeddables that are allowed to be selected by an author when embeddables are not disabled.
+
+### Edit Dialog Properties
+The following JCR properties are used:
+
+1. `./type` - defines the input type to use. Types include URL, embeddable and HTML.
+2. `./url` - defines the URL of the widget to embed.
+3. `./embeddableResourceType` - defines the resource type of an embeddable.
+4. `./html` - defines a HTML string to embed.
+
+## BEM Description
+```
+BLOCK cmp-embed
+```
+
+## Extending the Embed Component
+
+Extension of the Embed component is possible by providing any of the following:
+
+### Custom URL Processor
+
+By implementing the [Embed.UrlProcessor](../../../../../../../../../../../bundles/core/src/main/java/com/adobe/cq/wcm/core/components/models/embed/Embed.java) interface you can create your own URL processor.
+
+You will also need to create an HTL template file, with the same name as the `processor` field returned in the `Result`.
```

Comment:
It would be valuable to add a security considerations section in this documentation.
- only implement processor for trusted sources. The payload returned by the endpoint may contain javascript
- always fetch resources over HTTPS, without sslRelax set to true
- stick to HTL safe context whenever possible

Reply:
@acollign great idea, added in https://github.com/adobe/aem-core-wcm-components/pull/718/commits/a39a908375996fc254f864fcd78ca1fc668b6363